chain/cosmos/sidecar.go

```diff
@@ -93,10 +93,10 @@ func NewSidecar(
 // on a per validator level.
 func (s *SidecarProcess) Name() string {
 	if s.validatorProcess {
-		return fmt.Sprintf("%s-%s-%s-val-%d-%s", dockerutil.ICTDockerPrefix, s.Chain.Config().ChainID, s.ProcessName, s.Index, dockerutil.SanitizeContainerName(s.TestName))
+		return fmt.Sprintf("-%s-%s-val-%d-%s", s.Chain.Config().ChainID, s.ProcessName, s.Index, dockerutil.SanitizeContainerName(s.TestName))
```

Comment:
Was this leading '-' intentional?

Reply:
I specifically checked this 3 times and still missed it :cry: great catch ty

Reply:
9b096e47900a9d4684f2346c6be74d9935318368